core/src/main/kotlin/org/evomaster/core/output/service/HttpWsTestCaseWriter.kt

```diff
@@ -210,6 +212,24 @@ abstract class HttpWsTestCaseWriter : ApiTestCaseWriter() {
         lines: Lines,
         baseUrlOfSut: String
     ) {
+
+        val exActions = mutableListOf<HttpExternalServiceAction>()
+
+        // add all used external service actions for the action
+        if (TestWriterUtils.handleExternalService(config)) {
+            if (!format.isJavaOrKotlin()) {
+                log.warn("NOT support for other format ($format) except JavaOrKotlin")
```

Comment:
this would print this  log each time we do not run on JVM, which is not correct. here, a further check that is needed is to see if there is an ExternalAction. if there is no ExternalAction, no warning should be printed